Header header
float32 fwd_l
float32 fwd_r

IRArray l
IRArray r
